src/main/kotlin/paladin/core/service/organisation/OrganisationService.kt

```
@@ -0,0 +1,180 @@
+        // Delete the organisation itself
+        organisationRepository.delete(organisation)
+
+        logger.info { "Organisation with ID $organisationId deleted successfully." }
+    }
+
+    /**
+     * Invoked from Invitation accept action. Users cannot directly add others to an organisation.
+     */
+    fun addMemberToOrganisation() {
+
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)